templates/heroSlide.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="carousel component">
+  <div class="slider">
+    <div class="slider__component">
+      <div class="slider__content">
+        <h2 class="slider__title">{{title}}</h2>
+        <p class="slider__paragraph">{{description}}</p>
+      </div>
+      <div class="slider__img">
+        <img class="slider__image" src="{{{assetSrc image}}}" alt="{{altText}}">
+      </div>
+    </div>  
+    <div class="slider__component">
+      <div class="slider__content">
+        <h2 class="slider__title">{{title}}</h2>
+        <p class="slider__paragraph">{{description}}</p>
+      </div>
+      <div class="slider__img">
+        <img class="slider__image" src="{{{assetSrc image}}}" alt="{{altText}}">
+      </div>
+    </div>
+  </div>
+  <div class="dots-list">
+    <button class="dot-button active"></button>
+    <button class="dot-button"></button>
+  </div>
+</div>
+
```

Comment:
The template `heroSlide.hbs` should be only used for one slide. The logic for the carousel should be in a new remplate called `hero.hbs`. Use the `{{#each ..}}` expression along with the helper `{{{module ...}}}` to render the slides.